recipes/configurable-http-proxy/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: configurable-http-proxy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/jupyterhub/configurable-http-proxy.git
+  git_tag: 57fe19e0700c8bd3af0f77d8b7f51a54aa02fc87
+
+build:
+  number: 0
+  script: npm install -g .
```

Comment:
It is a brave new world... and so conda-forge begins packaging node packages 🎉 

Out of interest, have you had a look at the directory structure, and where the global directory actually is with this build? (I will take a look myself in the morning, but I'm just curious)